client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -100,6 +100,7 @@ public static FileSystem getHdfsFs(CelebornConf conf) {
       synchronized (ShuffleClient.class) {
         if (null == hdfsFs) {
           Configuration hdfsConfiguration = new Configuration();
+          hdfsConfiguration.set("fs.hdfs.impl.disable.cache", "false");
```

Comment:
suggest add some comments for this configuration

Reply:
OK.